cypress/fixtures/defaultUser.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Why make this a JSON file rather than a const variable?

Reply:
This allows you to import it from multiple places, without needing to share code. It's one of the things that the cypress docs recommend.

Reply:
I'm confused; how does an ordinary import not do that?